### PR TITLE
Remove-examples

### PR DESCRIPTION
**Pull request by ahms5**
### Changes proposed in this pull request:

- remove examples
- change despription in readme accoringly, ref to the gallery

open question: merge to main or develop?

### CODE REVIEW

**Comment by f-brinkmann** [comment]
Should we wait for https://github.com/pyfar/gallery/pull/51 with this? In this case, we could directly get rid of contributing as well and update the readme only once?

File: README.rst [hunk 1]
```diff
@@ -20,9 +20,9 @@ functions for digital audio signal processing.
 Getting Started
 ===============
 
-Check out the `examples notebook`_ for a tour of the most important pyfar
-functionality and `read the docs`_ for the complete documentation. Packages
-related to pyfar are listed at `pyfar.org`_.
+Check out the `pyfar.org`_ for a tour trought the pyfar
```
Comment (pingelit) [inline README.rst:23]:
```suggestion
Check out `pyfar.org`_ for a tour through the pyfar
```

**Review by pingelit — CHANGES_REQUESTED** [review]
Thanks for updating this.

One minor language change. Then I approve.

**Comment by mberz** [comment]
I'd suggest merging to main.

**Comment by ahms5** [comment]
replaced by #574 to merge to main

**Pull request closed without merging**